specs/Datepicker.md

```diff
@@ -40,7 +40,7 @@
 | selectedDate     | Date                                             | Date shown as selected in the `input`.                                                             |
 | shortDays        | string[]                                         | An array of localized strings for the short names of days.                                         |
 | shortMonths      | string[]                                         | An array of localized strings for the short names of months                                        |
-| type             | enum`day|month|year`                             | Type of the `Calendar` to be shown.                                                                |
+| type             | enum`day                                         | month                                                                                              | year`                                      | Type of the `Calendar` to be shown. |
```

Comment:
The change in line 23 is happening because of this, this should probably be:

```suggestion
| type             | enum`day \| month \| year`                                      | Type of the `Calendar` to be shown. |
```